templates/kubernetes/charmed-k8s.html

```diff
@@ -119,7 +119,7 @@ <h3 class="p-heading--4">Streamlined cluster lifecycle management with Charmed O
   </div>
   <div class="row">
     <div class="col-12">
-      <p><a href="https://juju.is/">Learn more about the Charmed Operator Framework&nbsp;&rsaquo;</a></p>
+      <p><a href="https://juju.is/">Learn more about Juju, software operators orchestration engine&nbsp;&rsaquo;</a></p>
```

Comment:
The copy looks different on the copy doc:
```suggestion
      <p><a href="https://juju.is/">Learn more about Juju, an orchestration engine for software operators&nbsp;&rsaquo;</a></p>
```

Reply:
Thanks for the review @lorumic! I added a comment in the copy doc for stakeholder's approval. So let's wait for his feedback on this.